frontends/mit-learn/src/pages/HomePage/SearchInput.tsx

```diff
@@ -1,8 +1,40 @@
 import React, { useCallback } from "react"
 import { RiSearch2Line, RiCloseLine } from "@remixicon/react"
-import { Input, AdornmentButton } from "ol-components"
+import { Input, AdornmentButton, styled, pxToRem } from "ol-components"
 import type { InputProps } from "ol-components"
 
+const StyledInput = styled(Input)(({ theme }) => ({
+  height: "72px",
+  "&.MuiInputBase-adornedEnd": {
+    paddingRight: "0 !important",
+    button: {
+      paddingRight: "10px",
+    },
```

Comment:
**Request / Question**  Do we need this? What is it for? I think the `width` below should be enough.